data/Its-Aman-Yadav.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "Aman Yadav",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Undergrad at KCCITM, just exploring this world :)",
```

Comment:
you are missing your avatar.  See docs here step 9 https://linkfree.eddiehub.io/docs/quickstart